### ISSUE TITLE
High contrast switch in `dark-www` and `editor-dark-mode`

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#5895]
### Why this would be helpful

Scratch is currently working on improving the entire site's contrast, but right now I wonder if we could come into play a little early and make our own high-contrast theme.

### My suggestion

We could bundle a bunch of color changes affecting many parts of the website that would be too hard for the average user to set all themselves into one switch called "high contrast colors" for the customizable website colors addon and maybe the editor one too. It would be meant to improve text and UI visibility for those with disabilities affecting vision. Modified elements would include button backgrounds, the message badge background, light-gray text, borders, icons, on/off switches and anything else I missed that's not already customizable.

If this was turned off, it could also revert color-related changes that have already happened, such as the darkening of the remix button. Or maybe we could skip that and just make it so that when this switch is off, nothing changes. This would save us some time and have the advantage of allowing users to get used to the changes and letting people like me who want something closer to vanilla Scratch not have to make any changes.

This could also be a three-way setting. One would revert contrast improvements, one wouldn't change anything, and one would apply a high-contrast theme.

We'd also have high contrast presets to go along with this.

### Possible alternatives

Or we could just wait for/help the Scratch Team to do it. Hopefully it won't be too much longer?

### Additional context

Related: #5539

**Comment by WorldLanguages** [comment]
If we add a way to revert contrast improvements, it should be a three-way setting, like you mentioned last. An on/off switch would be problematic because both options would change UI elements and we couldn't offer a "vanilla colors" preset anymore.

I don't know if we should or want to maintain a high contrast preset. It's definitely not my area of specialty. I'd wait to hear what mxmou thinks.

**Comment by Secret-chest** [comment]
Some coulours such as the orange should probably customisable by themselves.

**Comment by DNin01** [comment]
> Some coulours such as the orange should probably customisable by themselves.

Sure, there's probably a decent number of users who would want to change that.
